src/pages/MainPage/MainPage.tsx

```diff
@@ -33,6 +33,9 @@ export const MainPage: React.FC = () => {
     let boxIdx = manIdx;
     let boxCounter = 0;
     setGameRunning(true);
+    await new Promise((r) => setTimeout(r, cfg.intervalDur * 2));
```

Comment:
Extract this out as a util function for delay or sleep like below and use it everywhere
```ts
    const sleep = async (delayInMs: number) => new Promise((r) => setTimeout(r, delayInMs));
```